utils/eza/Makefile

```
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eza
+PKG_VERSION:=0.18.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/eza-community/eza/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=2200bc1c07511c105961dcb3fe1682cc9067d55280dac990de7623aff86cab1a
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:= rust/host
```

```suggestion
PKG_BUILD_DEPENDS:=rust/host
```

small nitpicking, nothing else, sorry!